src/perfetto_cmd/perfetto_cmd_android.cc

```diff
@@ -20,19 +20,28 @@
 
 #include <cinttypes>
 
-#include "perfetto/base/build_config.h"
 #include "perfetto/base/logging.h"
 #include "perfetto/ext/base/file_utils.h"
+#include "perfetto/ext/base/scoped_mmap.h"
 #include "perfetto/ext/base/string_utils.h"
 #include "perfetto/ext/base/uuid.h"
-#include "perfetto/tracing/core/trace_config.h"
+#include "perfetto/protozero/proto_decoder.h"
 #include "src/android_internal/incident_service.h"
 #include "src/android_internal/lazy_library_loader.h"
 #include "src/android_internal/tracing_service_proxy.h"
 
+#include "protos/perfetto/config/trace_config.gen.h"
+
+#include "protos/perfetto/trace/trace.pbzero.h"
+#include "protos/perfetto/trace/trace_packet.pbzero.h"
+
 namespace perfetto {
 namespace {
 
+// traced runs as 'user nobody' (AID_NOBODY) that is 9999.
+// We only trust packages written by traced.
```

Comment:
Can you please actually link to the file [1]

[1] https://cs.android.com/android/platform/superproject/+/android-latest-release:system/core/libcutils/include/private/android_filesystem_config.h;l=203